ui/components/src/test/tree/controlled/component/TreeRenderer.test.tsx

```diff
@@ -229,7 +229,8 @@ describe("TreeRenderer", () => {
     const verticalScrollSpy = sinon.spy();
     sinon.replace(VariableSizeList.prototype, "scrollToItem", verticalScrollSpy);
     const horizontalScrollSpy = sinon.spy();
-    sinon.replace(Element.prototype, "scrollIntoView", horizontalScrollSpy);
+    // sinon.replace(Element.prototype, "scrollIntoView", horizontalScrollSpy);
+    window.HTMLElement.prototype.scrollIntoView = horizontalScrollSpy;
 
```

Comment:
Why make this change? No the spy won't be cleaned up after the test.

Reply:
@roluk After adding iTwinUI-react, this horizontalScrollSpy test failed in a full package test or cover run. It still succeeded by itself, however. This is one change that would resolve the issue. I just added restoration of the scrollIntoView function. If you would like to assist in finding a better solution, please pull the branch.

Reply:
Placing the spy on `HTMLElement` prototype instead of `Element` seems to make all tests pass. Made the according changes in `2de52c134c8ebe421b60502ad285f7e66dfb9536`.